__tests__/test.js

```diff
@@ -1,13 +1,68 @@
 test.skip('Configuration options', () => {});
 
+const puppeteer = require('puppeteer');
+
 describe('Universal Viewer', () => {
+  let browser;
+  let page;
+
   beforeAll(async () => {
-    await page.goto('http://localhost:4444/');
+    browser = await puppeteer.launch();
+    page = await browser.newPage();
+    await page.goto('http://localhost:8080/#?xywh=-1741%2C125%2C8017%2C6807&iiifManifestId=http%3A%2F%2Fiiif.bodleian.ox.ac.uk%2Fexamples%2Fmushaf4.json');
+  });
+
+  afterAll(async () => {
+    await browser.close();
   });
+
   it('has the correct page title', async () => {
     const title = await page.title();
     expect(title).toBe('Universal Viewer Examples');
   });
+
+  it('extends thumbnail labels when "extend labels" checkbox is clicked', async () => {
+    
+    jest.setTimeout(10000);

```

Comment:
I see that you set testTimeout to 10000 in package.json, and you also set it here. Are both necessary?